src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/CSharpTestLspServerHelpers.cs

```diff
@@ -31,7 +33,8 @@ internal class CSharpTestLspServerHelpers
             };
 
             var exportProvider = RoslynTestCompositions.Roslyn.ExportProviderFactory.CreateExportProvider();
-            var workspace = CreateCSharpTestWorkspace(files, exportProvider, razorSpanMappingService);
+            var metadataReferences = await ReferenceAssemblies.Default.ResolveAsync(language: LanguageNames.CSharp, CancellationToken.None).ConfigureAwait(false);
```

Comment:
This is the main change (thanks to Sam for helping with the correct approach here!)